src/views/zesty/ABLandingPage.js

```diff
@@ -85,7 +85,7 @@ function ABLandingPage({ content }) {
 
   useEffect(() => {
     const filteredContent = content.datasets.data.filter(
```

Comment:
can we add conditional chaining in line 87?

Reply:
DOne :)